lib/components/viewers/stop-viewer.js

```diff
@@ -368,7 +367,17 @@ class StopViewer extends Component {
           >
             <Icon Icon={Search} style={{ marginLeft: '0.2em' }} />
           </button>
-          {!isFlex && (
+          <button
+            className="link-button pull-right"
+            onClick={this._viewNearby}
+            style={{ fontSize: 'small' }}
+          >
+            {/* FIXME: What icon should we use? */}
+            <IconWithText Icon={Calendar}>
+              <FormattedMessage id="components.StopViewer.viewNearby" />
+            </IconWithText>
+          </button>
+          {/* {!isFlex && (
```

Comment:
Do we still need the block below?

Reply:
I feel weird removing it without removing all the code that made the stop viewer a stop viewer, although we are now just using this for schedule viewing until we can make a new stop schedule viewer. Should I rip out all the code for the old stop viewer now that there's no way to access it?

Reply:
It will be deleted at some point when this file gets deleted entirely.